From 31cb742a740e95f24e457c12ffd83a5e0beef2fa Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megous@megous.com>
Date: Tue, 20 Jul 2021 21:08:22 +0200
Subject: [PATCH 388/446] arm64:dts:sun50i-h5: Disable idle states

Disable idle states on H5, until I can use crust
Otherwise the H5 boards just overheat doing nothing.

Signed-off-by: Ondrej Jirman <megous@megous.com>
---
 arch/arm64/boot/dts/allwinner/sun50i-h5.dtsi | 11 ++++++-----
 1 file changed, 6 insertions(+), 5 deletions(-)

diff --git a/arch/arm64/boot/dts/allwinner/sun50i-h5.dtsi b/arch/arm64/boot/dts/allwinner/sun50i-h5.dtsi
index 1952ac7ad..2fbc2ef56 100644
--- a/arch/arm64/boot/dts/allwinner/sun50i-h5.dtsi
+++ b/arch/arm64/boot/dts/allwinner/sun50i-h5.dtsi
@@ -18,7 +18,7 @@ cpu0: cpu@0 {
 			clocks = <&ccu CLK_CPUX>;
 			clock-latency-ns = <244144>; /* 8 32k periods */
 			#cooling-cells = <2>;
-			cpu-idle-states = <&cpu_sleep>, <&cluster_sleep>;
+//			cpu-idle-states = <&cpu_sleep>, <&cluster_sleep>;
 		};
 
 		cpu1: cpu@1 {
@@ -29,7 +29,7 @@ cpu1: cpu@1 {
 			clocks = <&ccu CLK_CPUX>;
 			clock-latency-ns = <244144>; /* 8 32k periods */
 			#cooling-cells = <2>;
-			cpu-idle-states = <&cpu_sleep>, <&cluster_sleep>;
+//			cpu-idle-states = <&cpu_sleep>, <&cluster_sleep>;
 		};
 
 		cpu2: cpu@2 {
@@ -40,7 +40,7 @@ cpu2: cpu@2 {
 			clocks = <&ccu CLK_CPUX>;
 			clock-latency-ns = <244144>; /* 8 32k periods */
 			#cooling-cells = <2>;
-			cpu-idle-states = <&cpu_sleep>, <&cluster_sleep>;
+//			cpu-idle-states = <&cpu_sleep>, <&cluster_sleep>;
 		};
 
 		cpu3: cpu@3 {
@@ -51,9 +51,9 @@ cpu3: cpu@3 {
 			clocks = <&ccu CLK_CPUX>;
 			clock-latency-ns = <244144>; /* 8 32k periods */
 			#cooling-cells = <2>;
-			cpu-idle-states = <&cpu_sleep>, <&cluster_sleep>;
+//			cpu-idle-states = <&cpu_sleep>, <&cluster_sleep>;
 		};
-
+/*
 		idle-states {
 			entry-method = "psci";
 
@@ -75,6 +75,7 @@ cluster_sleep: cluster-sleep {
 				arm,psci-suspend-param = <0x01010013>;
 			};
 		};
+*/
 	};
 
 	pmu {
-- 
2.31.1

